applications/FluidDynamicsApplication/python_scripts/apply_mass_conservation_check_process.py

```diff
@@ -46,20 +75,98 @@ def ExecuteInitialize(self):
 
         KratosMultiphysics.Logger.PrintInfo("ApplyMassConservationCheckProcess","Initialization finished (initial volumes calculated).")
 
-
-    def ExecuteBeforeSolutionLoop(self):
-        self.mass_conservation_check_process.ExecuteBeforeSolutionLoop()
-
-
-    def ExecuteInitializeSolutionStep(self):
-        self.mass_conservation_check_process.ExecuteInitializeSolutionStep()
-
+    def Execute(self):
```

Comment:
is this really necessary?

Reply:
hmm im afraid so. In a two fluid calculation, you usually want to correct your mass after doing the convection but before solving the fluid itself. Thats why you need an Execute method to be able to use this process not only as one process more in the processes list, but also to have it inside you two_fluids_solver and call it whenever you want.

Reply:
but how do you get this object out of the list of processes?

Reply:
The point is that this kind of processes are not understood to be in the list of processes but encapsulated inside the solver (same happens with the level set correction for the embedded CFD).

Aside of the formulatxion reasoning @ddiezrod pointed above, you don't want to completely expose this to the standard user. I think it's much more convenient and reasonable to control this kind of features through the `solver_settings`.